### PR TITLE
Design calculator UI

### PR DESCRIPTION
**Pull request by shafiuyushawu**
# **Features updated ✍️**

### 1. Created interface design for calculator UI

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          <div className="btn text-xl w-[5%] bg-slate-200">4</div>
+          <div className="btn text-xl w-[5%] bg-slate-200">5</div>
+          <div className="btn text-xl w-[5%] bg-slate-200">6</div>
+          <div className="btn text-xl w-[5%] bg-orange-400">-</div>
+        </div>
+        <div>
+          <div className="btn text-xl w-[5%] bg-slate-200">1</div>
+          <div className="btn text-xl w-[5%] bg-slate-200">2</div>
+          <div className="btn text-xl w-[5%] bg-slate-200">3</div>
+          <div className="btn text-xl w-[5%] bg-orange-400">+</div>
```
Comment (mikethreels) [inline src/components/Calculator.jsx]: - [x] according to the requirements Calculator.js should have at least 1 child component

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Shafiu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
- [x] there is still some boilerplate that got created by the CRA which should be deleted, in the public folder only the index.html file has to stay and in the src folder the test files and the Webvitals file can be removed as well.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+function Calculator() {
+  return (
+    <>
+      <div className="text-xl flex flex-col justify-center p-10">
+        <CalculatorScreen />
+        <div>
+          <div className="btn text-xl w-[5%] bg-slate-200">AC</div>
+          <div className="btn text-xl w-[5%] bg-slate-200">+/-</div>
+          <div className="btn text-xl w-[5%] bg-slate-200">%</div>
+          <div className="btn text-xl w-[5%] bg-orange-400">/</div>
```
Comment (julie-ify) [inline src/components/Calculator.jsx:12]: - [x] **[OPTIONAL]** I would recommend you use the correct sign: &div; according to the design.

**Review by julie-ify — APPROVED** [review]
Hi @shafiuyushawu,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shafiuyushawu** [comment]
@julie-ify 
Thank you

**Pull request merged**